packages/database/src/migrations/20210202031622-NutritionCounsellingTotalNewAndExistingClients-modifies-data.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+async function addReportToGroups(db, reportId, groupCodes) {
+  return db.runSql(`
+    UPDATE
+      "dashboardGroup"
+    SET
+      "dashboardReports" = "dashboardReports" || '{"${reportId}"}'
+    WHERE
+      "code" IN (${groupCodes.map(code => `'${code}'`).join(',')});
+  `);
+}
+async function removeReportFromGroups(db, reportId, groupCodes) {
+  return db.runSql(`
+    UPDATE
+      "dashboardGroup"
+    SET
+      "dashboardReports" = array_remove("dashboardReports", '${reportId}')
+    WHERE
+      "code" IN (${groupCodes.map(code => `'${code}'`).join(',')});
+  `);
+}
+async function deleteItemByOneCondition(db, table, condition) {
+  const [key, value] = Object.entries(condition)[0];
+  return db.runSql(`
+    DELETE FROM
+      "${table}"
+    WHERE
+      "${key}" = '${value}';
+  `);
+}
+const totalClientNumberIndicator = {
+  id: generateId(),
+  code: 'TO_HPU_Nutrition_Counselling_Number_Of_Total_Client',
```

Comment:
Similarly for the other indicators:

```suggestion
  code: 'TO_HPU_Nutrition_Counselling_Number_Of_Total_Clients',
```